File: Selenium/Activities/selenium_Activity1.java
package selenium_session1;

import org.openqa.selenium.*;
import org.openqa.selenium.firefox.FirefoxDriver;

public class selenium_Activity1 
{

	public static void main(String[] arg)
	{
		//create a WebDriver instance
		WebDriver driver = new FirefoxDriver();
		
		//go to a web page
		driver.get("https://training-support.net");
		
		//do stuff on the page
		System.out.println("Title of the page is : " + driver.getTitle());
		
		//Find the About Us link and click it
		//driver.findElement(By.linkText("About Us")).click();
		driver.findElement(By.xpath("/html/body/div/main/div/div/main/a")).click();
		
        // Print the page title of the About Us page
        System.out.println("New page title is: " + driver.getTitle());
		
		//close the browser
		driver.quit();
	}
}

File: Selenium/Activities/selenium_activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

#create WebDriver instance
with webdriver.Firefox() as driver:

    #go to web page
    driver.get("https://training-support.net")

    #do the stuff on the page
    pageTitle = driver.title
    print("Title of the page is : " + pageTitle)

    # Find the "About Us" button on the page using ID and click it
    #driver.find_element(By.LINK_TEXT, "About Us").click()
    driver.find_element(By.XPATH, "/html/body/div/main/div/div/main/a").click()

    # Print the title of the new page
    pageTitle = driver.title
    print("New page title is: ", pageTitle)
    #close the browser
    driver.quit()